db/migrate/20220223231534_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[6.1]
+  def change
+    create_table :posts do |t|
+      t.string :title
```

Comment:
- [ ] Let's look at the user and post relationship properly, the id of a user should exist in the post table because a post belongs to a user or author.

Reply:
Hey, thank you for asking, when we look at the ERD, you will see that it has authorId as a column name, this is where the relationship between a user(author) and a post is established.

![post](https://user-images.githubusercontent.com/53124548/155506773-9cb45349-d46f-445e-bafd-b49dc99f1a9a.png)

Reply:
We can also take a look at the schema, you added post_id instead of author_id

Reply:
Thanks for the explanation, fixed

Reply:
If you find my review not clear, you are free to get into a zoom call with me anytime. My slack username: elisha kyakopo

Reply:
It's good that you now understand, thank you so much👍